src/sql/workbench/services/objectExplorer/browser/asyncServerTreeRenderer.ts

```diff
@@ -43,6 +43,8 @@ class ConnectionProfileGroupTemplate extends Disposable {
 	set(element: ConnectionProfileGroup) {
 		let rowElement = findParentElement(this._root, 'monaco-list-row');
 		if (rowElement) {
+			// white is the color for connection profile group regardless themes in object explorer.
+			rowElement.style.color = 'white';
```

Comment:
Where is this from? Would be good to explain why since it seems kind of odd to hardcode something like this

Reply:
https://github.com/microsoft/azuredatastudio/blob/7b54abbc96f62cfbbb4c204a0f22ada581994bd5/src/sql/workbench/contrib/dataExplorer/browser/media/connectionViewletPanel.css#L45

I'll add a comment

Reply:
ideally, the connection group color should be defined as a pair, background-color and color so that we know they are contrast enough. but that is not in the scope of this PR, I will create an issue under Connection area to track this work.

Reply:
issue created: https://github.com/microsoft/azuredatastudio/issues/13138